apps/editors/templates/editors/includes/history.html

```diff
@@ -3,9 +3,15 @@
     {{ record.log.short }}
   </th>
   <td>
-    {% trans user = record.user|user_link, date = record.created|babel_datetime %}
-      <div>By {{ user }} on {{ date }}</div>
-    {% endtrans %}
+    {% if record.user %}
```

Comment:
Weird, I thought I'd seen `ActivityLog` entries with no user, there's none. So indeed i shouldn't do that. I'll use the `Mozilla` account. I guess you're talking about the user with email `addons-team@mozilla.com` (corresponding to `settings.LANGPACK_OWNER_EMAIL`)?